ckanext/scheming/validation.py

```diff
@@ -576,11 +576,12 @@ def comma_swap_formatter(arg_value):
     """
     if arg_value.count(',') == 1:
         parts = arg_value.split(',')
-        stripped_parts = map(lambda x: x.strip(), parts)
+        stripped_parts = map(str.strip(), parts)
```

Comment:
That's a bug. You need to pass a funtion to map - no parenthesis.
 `stripped_parts = map(str.strip, parts)`

Reply:
So this actually throws a bug because of the unicode/str encoding.  
```
TypeError: descriptor 'strip' requires a 'str' object but received a 'unicode' 
```
I'll leave as it is with the lambda function for now.